### ISSUE TITLE
Don't show a dark dot when a friend is offline

### ISSUE THREAD

**Issue by iphydf** [evilcorpltd/aTox#1186]
In bedtime mode, the screen is black and white and it's impossible to see whether the friend is online from that dot. Even without it, it's harder to see the colour than the presence of the dot. I suggest not showing it at all when offline.

**Comment by robinlinden** [comment]
Yeah, seems reasonable. I've heard from others that it can be hard to see when a contact is offline, even without the bedtime mode active, so some UX work here would be good.

**Comment by softwaretoxchat-crypto** [comment]
Yeah I saw it.
